sgrep/lib/parse_rules.ml

```diff
@@ -16,16 +16,21 @@ open Common
 
 module R = Rule
 
+exception InvalidRuleException of string * string
```

Comment:
I think you can remove the 'Exception' suffix, it's clear from the context that it's an exception.

Reply:
Yes, it's just a habit from Python